daemon/api_test.go

```diff
@@ -2385,3 +2385,89 @@ func (s *apiSuite) TestGetEvents(c *check.C) {
 	ts.Close()
 	c.Assert(d.hub.SubscriberCount(), check.Equals, 1)
 }
+
+func setupChanges(st *state.State) {
+	chg1 := st.NewChange("install", "install...")
+	t1 := st.NewTask("download", "1...")
+	t2 := st.NewTask("inst", "2...")
+	chg1.AddAll(state.NewTaskSet(t1, t2))
+	t1.Logf("l11")
+	t1.Logf("l12")
+	chg2 := st.NewChange("remove", "remove..")
+	t3 := st.NewTask("rm", "1...")
```

Comment:
s/inst/activate/
s/rm/unlink/

This is just an example, so easier to grasp if we don't repeat the terms.